src/app/app.component.html

```diff
@@ -4,11 +4,12 @@ <h1 class="intro">
   Welcome to {{title}}!
 </h1>
 
-<app-button-component routerLink="/foo">Navigate to "/foo"</app-button-component>
-<app-button-component routerLink="/bar/this_is_parameter_value">Navigate to "/bar"</app-button-component>
-<app-button-component routerLink="/qweqwe">Navigate to nonexistant page</app-button-component>
```

Comment:
Can we leave a link that will somehow navigate nowhere and end up 404, just for example purposes?

Reply:
Done